File: aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-vb/samples/sample17.vb
Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load 
 If Not Page.IsPostBack Then 
 If Session("MyMasterPage") IsNot Nothing Then 
 Dim li As ListItem = MasterPageChoice.Items.FindByText(Session("MyMasterPage").ToString())
 If li IsNot Nothing Then 
 li.Selected = True
 End If 
 End If 
 End If 
End Sub

File: aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-cs/samples/sample17.cs
protected void Page_Load(object sender, EventArgs e) 
{ 
	if (!Page.IsPostBack) 
	{ 
		if (Session["MyMasterPage"] != null)
		{
			ListItem li = MasterPageChoice.Items.FindByText(Session["MyMasterPage"].ToString());
			if (li != null) 
				li.Selected = true; 
		} 
	}
}